doc/modules/linear_model.rst

```diff
@@ -786,8 +786,13 @@ non-smooth `penalty="l1"`. This is therefore the solver of choice for sparse
 multinomial logistic regression. It is also the only solver that supports
 `penalty="elasticnet"`.
 
-In a nutshell, the following table summarizes the penalties supported by
-each solver:
+The "lbfgs" is an optimization algorithm that approximates the 'BFGS'_, algortithm
```

Comment:
I don't know what `'BFGS'_` is meant to do. It is unfamiliar restructured text syntax

Reply:
Let me update for BFGS.